### PR TITLE
PR to add the configs/rhel8lab tree for the Definitive RHEL8 Hands On Lab

### PR DESCRIPTION
**Pull request by gmkeegan**
##### SUMMARY

First PR to add configs/rhel8lab to AgnosticD

##### ISSUE TYPE

- New config Pull Request

##### COMPONENT NAME

rhel8lab, Definitive RHEL8 Hands On Lab, T78321

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: ansible/configs/rhel8lab/files/cloud_providers/ec2_cloud_template.j2 [hunk 1]
```diff
@@ -0,0 +1,287 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Mappings:
```
Comment (stencell) [inline ansible/configs/rhel8lab/files/cloud_providers/ec2_cloud_template.j2:2]: Consider using something like this here: https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles-infra/infra-ec2-template-generate/templates/cloud_template.j2#L5

That way you can take advantage of this healthier list (including cheaper GOLD images) that doesn't have to be maintained in a separate config and get stale.

ami_mappings are in: https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles-infra/infra-ec2-template-generate/defaults/main.yml

**Comment by wkulhanek** [comment]
@gmkeegan It might be your requirements file. You have nothing in it (except the --- on the top). Try removing that file from your PR.

**Comment by tonykay** [comment]
@gmkeegan we are blocked on on-boarding until we can merge, thanks. I'll reach out.

**Comment by NonyNo3lle** [comment]
DO NOT MERGE - Tony will contact developer, and Mitesh will be involved to provide some recommendations.

**Comment by wkulhanek** [comment]
@gmkeegan Is this PR still necessary?

**Comment by NonyNo3lle** [comment]
Closing, no longer needed.

**Pull request closed without merging**